docs/openapi.yaml

```diff
@@ -0,0 +1,1143 @@
+              schema:
+                $ref: "#/components/schemas/Mission"
+          description: Request successful
+        "404":
+          description: Not found - Mission object does not exist
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+  /missions/{mission_id}/reports:
+    get:
+      operationId: getSingleMissionReports
+      summary: Lookup all reports for mission
+      description: |
+        ### Overview
+        Get all reports of missions with the given mission id.
+      tags:
+        - Missions
+      parameters:
+        - in: path
+          name: mission_id
+          required: true
+          schema:
+            type: integer
+            example: 271
+        - in: query
+          name: startTime
+          schema:
+            type: string
+            format: date-time
+            description: Minimum start time for returned reports
+          required: false
+        - in: query
+          name: endTime
+          schema:
+            type: string
+            format: date-time
+            description: Maximum end time for returned reports
+          required: false
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Report"
+          description: Request successful
+        "404":
+          description: Not found - Mission object does not exist
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+  /maps:
+    get:
+      operationId: getMaps
+      summary: Lookup all maps for asset
+      description: |
+        ### Overview
+        Lookup all maps for asset
+      tags:
+        - Maps
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Map"
+          description: Request successful
+        "404":
+          description: Not found - No map obejcts exist
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+
+  /maps/{map_id}:
+    get:
+      operationId: getSingleMap
```

Comment:
we have `getSingleMap`, similarly to `getSingleRobot`. Do we need to use the word single? What about `getMap` or `getSpecifiedMap`?